solution/index.js

```diff
@@ -0,0 +1,222 @@
+let Tasks = {
+    'todo': [],
+    'in-progress': [],
+    'done': []
+};
+
+// maping the ul names to make genrated code 
+const mapper = {
+    '1': 'todo',
+    '2': 'in-progress',
+    '3': 'done'
+}
+
+let mouseHover;
+let onAlt = false;
```

Comment:
Don't use global variables. They only cause trouble. Find a different way to do this.